test/DartTest.ts

```diff
@@ -275,6 +277,10 @@ describe('transpile to dart', () => {
       expectTranslate("`\\$a`").to.equal(" '''\\$a''' ;");
     });
 
+    it('escapes escape sequences', () => {
+      expectTranslate("`\\\\u1234`").to.equal(" '''\\\\u1234''' ;");
```

Comment:
Yes, I believe you now. The extra double quotes around the string are the clue.
An e2e test would be easier to read in this respect, though I think this edge case doesn't need such a heavyweight verification.